10.Mender-server/docs.md

```diff
@@ -7,6 +7,60 @@ shortcode-core:
 github: false
 ---
 
+
+## Mender 3.7.7
+
+_Released 08.15.2024_
```

Comment:
:no_good_man: 
```suggestion
_Released 15.08.2024_
```

Reply:
I am sorry, we have been using american dates for 6 years. Changing this now will make the page inconsistent.

But let's use ISO dates with the new release process :pray: